internal/pkg/scaffold/go_mod.go

```diff
@@ -73,16 +55,17 @@ replace (
 	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver kubernetes-1.14.1
 	k8s.io/apimachinery => k8s.io/apimachinery kubernetes-1.14.1
 	k8s.io/client-go => k8s.io/client-go kubernetes-1.14.1
+	k8s.io/cloud-provider => k8s.io/cloud-provider kubernetes-1.14.1
 )
 
 replace (
 	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.31.1
+	k8s.io/kube-state-metrics => k8s.io/kube-state-metrics v1.6.0
```

Comment:
Weird. I swear I added this comment to the review... Anyways. Doesn't there need to be a matching `require` for every `replace`ed module. Not sure this `replace` by itself does anything.

Reply:
Correct, and in fact this isn't used in operator projects, just in the SDK itself. Removing.